_platforms/en/spid.md

```diff
@@ -196,7 +196,8 @@ SPID can be integrated in the websites of the Public Administration, but also on
 2. Use and contribute to the open source components available in [Developers Italia](https://github.com/italia?q=spid).
 3. Use a test identity provider such as[spid-saml-check](https://github.com/italia/spid-saml-check){:target="_blank"} or [spid-testenv2](https://github.com/italia/spid-testenv2){:target="_blank"} to simulate the authentication flow and verify that your implementation is correct.
 4. Get in touch with other developers at [Slack](https://slack.developers.italia.it/){:target="_blank"}.
-5. Follow the accreditation procedure [described here](https://www.spid.gov.it/come-diventare-fornitore-di-servizi-pubblici-e-privati-con-spid){:target="_blank"}.
+5. Follow the onboarding [described here](https://www.spid.gov.it/come-diventare-fornitore-di-servizi-pubblici-e-privati-con-spid){:target="_blank"}.
+6. If you have any further questions or are having problems with onboarding, please consult the [SPID HelpDesk](https://helpdesk.spid.gov.it/index.php?a=add) page.
```

Comment:
I think "onboarding" can't be used as name

"Follow the onboarding procedure"
and
"[...] problems when onboarding"

sound better to me.

Reply:
that's right, done!

Reply:
done!